docs/source/examples/test_no_depends_fails.py

```diff
@@ -0,0 +1,18 @@
+from pych.extern import Chapel
+
+@Chapel(sfile="users.onlyonce.chpl")
+def useTwoModules(x=int, y=int):
+    return int
+
+if __name__ == "__main__":
+    print(useTwoModules(2, 4))
+
+import testcase
+# contains the general testing method, which allows us to gather output
+import os.path
+
+def test_using_multiple_modules():
+    out = testcase.runpy(os.path.realpath(__file__))
+    # Ensure that when a used module is nowhere near the exported function, we
+    # get an error message to that effect.
+    assert "error: Cannot find module \'M1\'" in out
```

Comment:
No need to change this, but fyi single quotes don't need to be escaped in a double quoted strings (and vice versa). E.g. these are valid:

```python
with_double = "something 'quoted'"
with_single = 'something "quoted"'